test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -143,6 +188,26 @@ private void CompareWorkspacesDifferentGuids(WorkspaceComparisonData a, Workspac
                 Assert.AreEqual(countA, countB, string.Format("One {0} node has {1} outports, while the other has {2}", a.NodeTypeMap[kvp.Key], countA, countB));
             }
 
+            foreach (var portkvp in a.PortDataMap)
+            {
+                //convert the old guid to the new guid
+                var newGuid = GuidUtility.Create(GuidUtility.UrlNamespace, this.modelsGuidToIdMap[portkvp.Key]);
+                Assert.IsTrue(b.PortDataMap.ContainsKey(newGuid));
+                var aPort = a.PortDataMap[portkvp.Key];
+                var bPort= b.PortDataMap[newGuid];
+                Assert.AreEqual(aPort.UseLevels,bPort.UseLevels);
+                Assert.AreEqual(aPort.KeepListStructure, bPort.KeepListStructure);
+                Assert.AreEqual(aPort.Level, bPort.Level);
+            }
+
+            foreach(var kvp in a.NodeReplicationMap)
```

Comment:
You added spaces after the "foreach" for all (most) of the other "foreach" statements, space here?